rlogin/src/components/SendTransaction.jsx

```diff
@@ -0,0 +1,63 @@
+import { useContext, useState } from 'react';
+import rLoginContext from '../contexts/rLoginContext';
+import { rskTestnet } from '../rskConfig';
+
+const faucetAddress = '0x88250f772101179a4ecfaa4b92a983676a3ce445';
+
+function SendTransaction() {
+  const { provider, account, tryRequest } = useContext(rLoginContext);
+
+  const [toAddress, setToaddress] = useState(faucetAddress);
```

Comment:
```suggestion
  const [toAddress, setToAddress] = useState(faucetAddress);
```
camelCasing for clarity - change other occurrences too